services/hosted_services.md

```diff
@@ -0,0 +1,19 @@
+# Hosted Systems in Use at PUL
+
+## tigris.princeton.edu
+
+### Purpose
+Records management software used by both University staff and Library staff in the Library's Records Management. Anne-Marie Phillips is the primary contact for the software. The software product name for this is "Infolinx". Further information available at https://aisww.com/infolinx/. 
+
+### Access
+Visit https://tigris.princeton.edu. Tigris is integrated with our campus IDP. Access to it is governed by membership in certain active directory works established specifically for tigris. These need to be managed via https://tools.princeton.edu. 
+
+Groups defined
+* TIGRIS Administrator 1 User (Anne Marie)
+* TIGRIS Legal User
+* TIGRIS Read Write
+* TIGRIS Storage User
+* TIGRIS User  
+
+### Renewing the SSL Cert
+Tigris.princeton.edu has an A DNS record pointing to: princetonuniversity.cloudapp.net. When the certificate is expiring a new one needs to be emailed to support for tigris at: support@gimmal.com. Must be submitted as a .pfx file. 
```

Comment:
Not a blocker, but I'm curious: how does gimmal.com relate to aisww.com? Are we leasing software from aisww and buying support from gimmal? or is one a subsidiary of the other or something?

Reply:
Aisww is the vendors website that the infolinx brand is attached to. https://gimmal.com/ appears to be another type of records management software. Maybe gimmal bought them out and provides support now.